### ISSUE TITLE
Can't archive Wikipedia:Sockpuppet investigations/Sd004973

### ISSUE THREAD

**Issue by roysmith** [GeneralNotability/spihelper#29]
Permalink:
https://en.wikipedia.org/w/index.php?title=Wikipedia:Sockpuppet_investigations/Sd004973&oldid=1010411385

When I do More / SPI-Archive, I get:

```
 Purged Wikipedia:Sockpuppet investigations/Sd004973
 Done!
```

but it doesn't actually archive the page.  If I go into More/SPI, "Archive case (Clerk only)" is greyed out.  I assume there's something broken in the markup, but I can't spot it.

**Comment by GeneralNotability** [comment]
@roysmith the problem was because the {{SPI case status}} template was, for whatever reason, above the date header. The "can we archive" came back true because there was a case status template that said closed _somewhere_ in the SPI, but when it iterated over the cases none of them had status "closed". I could look into adding a sanity check to warn if a case section is missing the case status template...

**Comment by roysmith** [comment]
Makes sense, thanks.  I think the real fix is to forbid people to hand-edit SPI cases, but I don't imagine I'll get much traction on that :-)

**Comment by Dreamy-Jazz** [comment]
Fixed by b4428a25cc7e2cdf9fa6277592e6869f7ad1c6a3

**Comment by roysmith** [comment]
The warning you added is an improvement, so I'm good.
